src/test/java/com/redhat/jenkins/plugins/ci/integration/FedMsgMessagingPluginIntegrationTest.java

```diff
@@ -409,6 +409,7 @@ public void testPipelineSendMsgReturnMessage() throws Exception {
         job.sandbox.check(true);
         job.save();
         job.startBuild().shouldSucceed();
+        assertThat(job.getLastBuild().getConsole(), containsString("\"timestamp\":1"));
```

Comment:
Are you sure the timestamp will begin with 1 all the time?

Reply:
Fixed